spec/requests/api/v1/articles_spec.rb

```diff
@@ -18,6 +18,29 @@
       get "/api/v1/articles", headers: headers
       expect(response.status).to eq 200
     end
+    
+    it "has a category key in the response" do
+      get "/api/v1/articles", headers: headers
+      expect(json_response.first).to include('category')
```

Comment:
can you test this with a loop like we do in cypress?

Reply:
done